src/ocaml_tools.ml

```diff
@@ -120,7 +120,10 @@ let ocamldoc_l_dir tags deps _docout docdir =
       Cmd (S[A"mkdir"; A"-p"; Px docdir]);
       Cmd (S [!Options.ocamldoc;
               S(List.map (fun a -> S[A"-load"; P a]) deps);
-              T(tags++"doc"++"docdir"); A"-d"; Px docdir])]
+              T(tags++"doc"++"docdir"); A"-d"; Px docdir]);
+      Cmd (S [A "ln"; A "-sf";
+              P (Pathname.concat !Options.build_dir docdir);
+              A Pathname.pwd])]
```

Comment:
I'm not sure I like the fact of spreading out the symbolic-linking logic in more places (there is already some in this file to handle `.itarget` rule). This aspect of ocamlbuild is currently doubtful (see #14 ). In particular, your implementation does not respect the `-no-links` option, so it is a regression on this front.

For now I think the simplest step forward would be to drop this part of the PR, and concentrate on good documentation targets.